### PR TITLE
Added error handling for PlistMuxConnection.get_device_list

### PR DESCRIPTION
**Pull request by AnNEDoMini**
Method get_device_list was lacking error handling

### CODE REVIEW

File: pymobiledevice3/usbmux.py [hunk 1]
```diff
@@ -353,7 +353,14 @@ def get_device_list(self, timeout: float = None) -> None:
         """ get device list synchronously without waiting the timeout """
         self.devices = []
         self._send({'MessageType': 'ListDevices'})
-        for response in self._receive(self._tag - 1)['DeviceList']:
+        ret = self._receive(self._tag - 1)
```
Comment (doronz88) [inline pymobiledevice3/usbmux.py]: let's simply replace with `_send_receive()`. just add a `return response` to avoid code duplication

**Review by doronz88 — CHANGES_REQUESTED** [review]

File: misc/pyinstaller.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Building Windows executable (exe)
```
Comment (doronz88) [inline misc/pyinstaller.md:1]: this shouldn't be part of this PR

**Review by doronz88 — CHANGES_REQUESTED** [review]
don't forget to fixup the second commit into the first one

File: misc/pyinstaller.md (hunk 2 above)
Comment (AnNEDoMini) [reply misc/pyinstaller.md:1]: Not sure how this happened. Will open a new one
